cross-chain/arbitrum/contracts/interfaces/IWETH.sol

```diff
@@ -0,0 +1,14 @@
+// contracts/Bridge.sol
+// SPDX-License-Identifier: Apache 2
+
+// Copy from https://github.com/wormhole-foundation/wormhole/blob/main/ethereum/contracts/bridge/interfaces/IWETH.sol
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface IWETH is IERC20 {
```

Comment:
Do we need that interface? Will we need to deal with WETH at all?

Reply:
There are a couple of things to unwrap here. First, I'm not even sure if we'll need these interfaces, that was my assumption at the time of writing because we don't have a generic Wormhole Gateway yet. But if it turns out that we need these interfaces, then [here](https://github.com/wormhole-foundation/wormhole/blob/main/ethereum/contracts/bridge/interfaces/ITokenBridge.sol#L129) there's a reference to `WETH`

Reply:
Ok clear!